$NetBSD: patch-pyproject.toml,v 1.1 2023/05/04 07:22:08 adam Exp $

pdm-pep517 as been renamed and re-published as pdm-backend.

--- pyproject.toml.orig	2023-05-04 07:10:54.000000000 +0000
+++ pyproject.toml
@@ -56,6 +56,6 @@ known_first_party = [
 
 [build-system]
 requires = [
-    "pdm-pep517",
+    "pdm-backend",
 ]
-build-backend = "pdm.pep517.api"
+build-backend = "pdm.backend"
